src/js/wrapper.ts

```diff
@@ -439,15 +456,31 @@ export const NATIVE: SentryNativeWrapper = {
    * @param key string
    * @param context key-value map
    */
-  setContext(key: string, context: { [key: string]: unknown } | null): void {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  setContext(key: string, context: { [key: string]: any } | null): void {
```

Comment:
l: The JSDoc comment above mentions `Not implemented in Android yet.`. Since `RNSentryModuleImpl` implements the `setContext` method I think this is not accurate.

Reply:
Yeah, thanks. That's not valid for a while.